docs/source/tutorials_source/package/tutorial_custom_augmentations.py

```diff
@@ -380,4 +441,6 @@ def plot_knn_multilabels(embeddings, multilabels, samples_idx, n_neighbors=50):
 # plot the distribution of the multilabels of the k nearest neighbors of 
 # the three example images at index 4111, 3340, 1796
 k = 20
```

Comment:
If you hardcode the sample indices anyway, you can also inline the `k=20` directly as `n_neighbors=20`

Reply:
True, done :)